integration/monai/examples/spleen_ct_segmentation/submit_job.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+echo "PYTHONPATH is ${PYTHONPATH}"
+
+algorithms_dir="${PWD}"
+
+config=$1
```

Comment:
suggest change `config=$1` to `job_configs=$1` or `job_folder=$1`

Reply:
thanks. I changed it.